hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/counters/FileSystemCounterGroup.java

```diff
@@ -311,8 +311,11 @@ public void readFields(DataInput in) throws IOException {
       String scheme = WritableUtils.readString(in); // scheme
       int numCounters = WritableUtils.readVInt(in); // #counter
       for (int j = 0; j < numCounters; ++j) {
-        findCounter(scheme, enums[WritableUtils.readVInt(in)])  // key
+        int countTypeIndex = WritableUtils.readVInt(in);
+        if(countTypeIndex < enums.length) {
```

Comment:
nit: can you add a space between if and (

Reply:
> nit: can you add a space between if and (

@steveloughran  
Thanks for review, fixed as you mentioned.

I have try to write a UT but it's not that easy to construct the scenario,
Just simple fix, I think it is ok without UT.

Pls help to merge once the pipeline is done,
Thanks a lot